cleanup/deleteByPropertyValue/DeleteByPropertyValueTest.groovy

```diff
@@ -39,4 +39,33 @@ class DeleteByPropertyValueTest extends Specification {
         cleanup:
         repo?.delete()
     }
+
+    def 'delete by property value dryRun test'() {
+
+        setup:
+        def baseurl = 'http://localhost:8088/artifactory'
+        def artifactory = ArtifactoryClientBuilder.create().setUrl(baseurl)
+            .setUsername('admin').setPassword('password').build()
+        def builder = artifactory.repositories().builders().localRepositoryBuilder()
+        builder.key('cleanup-local')
+        builder.repositorySettings(new GenericRepositorySettingsImpl())
+        artifactory.repositories().create(0, builder.build())
+        def repo = artifactory.repository('cleanup-local')
+        def somefile = new ByteArrayInputStream('content'.bytes)
+        repo.upload('somefile', somefile).withProperty('someprop', '4').doUpload()
+        def plugin = artifactory.plugins().execute('deleteByPropertyValue')
+        plugin.withParameter('propertyName', 'someprop')
+        plugin.withParameter('propertyValue', '5')
+        plugin.withParameter('repo', 'cleanup-local')
+        plugin.withParameter('dryRun', 'true').sync()
+
+        when:
+        repo.file('somefile').info()
+
+        then:
+        notThrown(HttpResponseException)
```

Comment:
To validate if the file is present, we need to check if the return of the `info()` call is not null.

Something like:

```
when:
def fileInfo = repo.file('somefile').info()

then:
fileInfo != null
```